wrappers/services.go

```diff
@@ -140,7 +140,7 @@ func stopService(sysd systemd.Systemd, app *snap.AppInfo, inter interacter) erro
 // StartServices starts service units for the applications from the snap which
 // are services. Service units will be started in the order provided by the
 // caller.
-func StartServices(apps []*snap.AppInfo, inter interacter, tm timings.Measurer) (err error) {
+func StartServices(apps []*snap.AppInfo, disabledSvcs []string, enableBeforeStart bool, inter interacter, tm timings.Measurer) (err error) {
```

Comment:
feels like we should turn  Stop/RestartFlags into Stop/RestartServicesFlags  and have also a StartServicesFlags